frontend/src/metabase-types/api/mocks/card.ts

```diff
@@ -10,29 +10,36 @@ import type {
   TableColumnOrderSetting,
 } from "metabase-types/api";
 
+// Do not import from "metabase/entities/collections" to avoid circular dependency
+import { ROOT_COLLECTION } from "metabase/entities/collections/constants";
```

Comment:
[commented below](https://github.com/metabase/metabase/pull/35137/files#r1377414161), if we're fine treating `card` as if the root collection would be null (while BE actually sends the root collection in card endpoints) then this could be removed.

Reply:
Fixed in 536582175599dcf0d9cd12880fc76eadcbdb3151